examples/README.md

```diff
@@ -40,13 +40,14 @@ git checkout v0.4.0
   - [Application](#application)
   - [Assets](#assets)
   - [Audio](#audio)
-  - [Diagnostics](#diagnostics)
+  - [Development](#development)
```

Comment:
I think this should probably continue to be called diagnostics. First: The `[Tests]` section is empty and I think `how_to_test_systems` belongs in a Tests section. Second, I don't see diagnostics and tests to be suitably similar to lump them under the same Development section.

Reply:
moved back to test! I'll come back on this point in a later PR probably with more arguments